group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/GeneralUniformAssignmentBuilderTest.java

```diff
@@ -271,45 +268,37 @@ public void testReassignmentForTwoMembersThreeTopicsGivenUnbalancedPrevAssignmen
             mkMapOfPartitionRacks(4)
         ));
 
-        Map<String, AssignmentMemberSpec> members = new TreeMap<>();
+        Map<String, MemberSubscriptionSpecImpl> members = new TreeMap<>();
+        Map<Uuid, Set<Integer>> targetAssignment;
 
-        Map<Uuid, Set<Integer>> currentAssignmentForA = new TreeMap<>(
-            mkAssignment(
-                mkTopicAssignment(topic1Uuid, 0, 1, 2)
-            )
+        targetAssignment = mkAssignment(
+            mkTopicAssignment(topic1Uuid, 0, 1, 2)
         );
```

Comment:
While we are doing those changes, I think that it would be better to directly inline the assignment and avoid the reused local variable `targetAssignment`.

Reply:
It looked too hectic when I inlined it so I tried to make it more readable

Reply:
```
        members.put(memberA, new MemberSubscriptionSpecImpl(
            Optional.empty(),
            Collections.singleton(topic1Uuid),
            new Assignment(mkAssignment(
               mkTopicAssignment(topic1Uuid, 0, 1, 2)
            ))
        ));
```

This does not seem to bad. What do you think?

Reply:
not with the one liners but when there's multiple topics it was very hectic

Reply:
It's okay I'll inline it if you feel like it's fine